### PR TITLE
[Flow] add types to SecondaryPanes/BreakpointsDropdown.js 

### PR DESCRIPTION
**Pull request by johnsonwe**
Fixes Issue: #5120 

Here's the Pull Request Doc
https://devtools-html.github.io/debugger.html/CONTRIBUTING.html#pull-requests

### Summary of Changes

* Added typings for BreakpointDropdown

### Test Plan

Tell us a little a bit about how you tested your patch.

Example test plan:

- [x] Command-P opens the panel
- [x] Clicking “+” opens the panel
- [x] Clicking a source navigates to the source
- [x] Clicking "x" closes the panel

Here's the Debugger's Testing doc
https://docs.google.com/document/d/1oBMRxV8A2ag2t22YsQOxTdEv0mXKzIg0tggJjRkU1S0/edit#.
Feel free to improve it!

### CODE REVIEW

**Comment by jasonLaster** [comment]
Please fix the title

File: src/components/SecondaryPanes/BreakpointsDropdown.js [hunk 1]
```diff
@@ -1,11 +1,12 @@
+// @flow
 import React from "react";
 import Svg from "../shared/Svg";
 import Dropdown from "../shared/Dropdown";
 import classnames from "classnames";
 
 import "./BreakpointsDropdown.css";
 
-function renderPause(isWaitingOnBreak) {
+function renderPause(isWaitingOnBreak: Object) {
```
Comment (AnshulMalik) [inline src/components/SecondaryPanes/BreakpointsDropdown.js]: `isWaitingOnBreak` is a `boolean`

File: src/components/SecondaryPanes/BreakpointsDropdown.js [hunk 2]
```diff
@@ -21,8 +22,8 @@ function renderPause(isWaitingOnBreak) {
 }
 
 function renderPauseOnExceptions(
-  shouldPauseOnExceptions,
-  shouldIgnoreCaughtExceptions
+  shouldPauseOnExceptions: Boolean,
+  shouldIgnoreCaughtExceptions: Boolean
```
Comment (AnshulMalik) [inline src/components/SecondaryPanes/BreakpointsDropdown.js]: Let's use `boolean` (small b) for consistency.

File: src/utils/ui.js [hunk 3]
```diff
@@ -14,7 +15,7 @@ export function isVisible() {
 
 /* Gets the line numbers width in the code editor
  */
-export function getLineNumberWidth(editor) {
+export function getLineNumberWidth(editor: Object) {
```
Comment (AnshulMalik) [inline src/utils/ui.js]: Let's stick to `BreakpointsDropdown.js` file in this PR, we can have another PR for this one :)

File: src/components/SecondaryPanes/BreakpointsDropdown.js [hunk 4]
```diff
@@ -80,16 +81,16 @@ function renderIgnoreExceptions(
...
 }
 
 export default function renderBreakpointsDropdown(
-  breakOnNext,
-  pauseOnExceptions,
-  shouldPauseOnExceptions,
-  shouldIgnoreCaughtExceptions,
-  isWaitingOnBreak
+  breakOnNext: Boolean,
+  pauseOnExceptions: Boolean,
```
Comment (AnshulMalik) [inline src/components/SecondaryPanes/BreakpointsDropdown.js]: `breakOnNext` and `pauseOnExceptions` should be `Function`

**Comment by darkwing** [comment]
Thank you for this @johnsonwe !  Unfortunately we can't merge this PR because `BreakpointsDropdown` has been removed for the new design; I'd love to see your change to `src/utils/ui.js` in another PR though! :)

**Pull request closed without merging**